cli/src/semgrep/app/scans.py

```diff
@@ -42,6 +42,13 @@
 logger = getLogger(__name__)
 
 
+@dataclass
+class ScanCompleteResult:
```

Comment:
I started to specify those in ATD here: https://github.com/returntocorp/semgrep-interfaces/pull/147/files#diff-d5b9d5c903db7121614125cc6d56a68f8f4867eb8e0f6a1569398d9f45c781c3R1104
It's in comment, but we should replace those dataclass by it.

Reply:
I'm not really sure it makes sense to use the the ATD here, maybe this deserves a better name.

This return is the only part of the servers response that needs to go outside the `scan_handler.report_findings` method, so the type here is internal to the cli implementation.